recipes/libuvc/all/patches/0.0.7-0001-fix-cci-packagename.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/a/CMakeLists.txt b/b/CMakeLists.txt
+index fbaffc9..e14daf4 100644
+--- a/a/CMakeLists.txt
++++ b/b/CMakeLists.txt
+@@ -116,7 +116,7 @@ foreach(target_name IN LISTS UVC_TARGETS)
+   )
+   target_link_libraries(${target_name}
+     # libusb-1.0 used internally so we link to it privately.
+-    PRIVATE LibUSB::LibUSB ${threads}
++    PRIVATE libusb::libusb ${threads}
```

Comment:
We could remove the patches and have CMakeDeps set the correct name in the generate method too 🙏